	Fix for xf86-video-siliconmotion in 16bpp with XAA, see:
	http://groups.google.com/group/loongson-dev/browse_thread/thread/570549e3d7894fef/5f04b3d9370adfd2
--- xorg-server-1.9.3.orig/fb/fb.h	2010-12-13 20:42:27.000000000 +0100
+++ xorg-server-1.9.3/fb/fb.h	2012-10-01 13:23:55.000000000 +0200
@@ -71,7 +71,7 @@
 #define FBPREFIX(x) fb##x
 #define WRITE(ptr, val) (*(ptr) = (val))
 #define READ(ptr) (*(ptr))
-#define MEMCPY_WRAPPED(dst, src, size) memcpy((dst), (src), (size))
+#define MEMCPY_WRAPPED(dst, src, size) memmove((dst), (src), (size))
 #define MEMSET_WRAPPED(dst, val, size) memset((dst), (val), (size))
 
 #endif
